src/nvim/help.c

```diff
@@ -817,6 +818,10 @@ void fix_help_buffer(void)
                 convert_setup(&vc, NULL, NULL);
 
                 ml_append(lnum, cp, (colnr_T)0, false);
+                // todo(clason): adjust extmarks and mark added line as changed to force reparsing
+                // of tree for treesitter
+                mark_adjust(lnum + 1, (linenr_T)MAXLNUM, (linenr_T)1, 0L, kExtmarkUndo);
```

Comment:
normally we do this after an entire change loop, like this https://github.com/neovim/neovim/compare/master...bfredl:neovim:nochange

Reply:
much better, thank you!